app/preview/preview.html

```diff
@@ -1,5 +1,6 @@
 <div class="preview-frame"><iframe ng-src="{{ url }}" name="preview-frame"></iframe></div>
 <p ng-if='file'>
-  Previewing {{ file.title }} <a ng-href="{{ url }}" download="filename">(Download)</a>
+  <span translate>Previewing {{ file.title }}</span>
+  <a ng-href="{{ url }}" download="filename">({{ "Download" | translate }})</a>
```

Comment:
Should the parens be include in the translation string? Eg Arabic would want them facing the other direction I think.  Can/should this be all one translation string?